Valet/VALSecureEnclaveValet.m

```diff
@@ -74,7 +74,7 @@ - (BOOL)canAccessKeychain;
 {
     // To avoid prompting the user for Touch ID or passcode, create a VALValet with our identifier and accessibility and ask it if it can access the keychain.
     VALValet *noPromptValet = nil;
-    if ([self isSharedAcrossApplications]) {
+    if (self.isSharedAcrossApplications) {
```

Comment:
non-blocking nit: if using dot syntax, use the property name rather than the getter. i.e. `self.sharedAcrossApplications`